tests/cross/src/test/scala/tests/pc/InlineValueSuite.scala

```diff
@@ -0,0 +1,168 @@
+package tests.pc
+
+import tests.BaseCodeActionSuite
+import munit.TestOptions
+import munit.Location
+
+import scala.meta.internal.metals.TextEdits
+import org.eclipse.{lsp4j => l}
+import scala.meta.internal.jdk.CollectionConverters._
+import scala.meta.internal.metals.CompilerRangeParams
+import java.net.URI
+import scala.meta.internal.mtags.CommonMtagsEnrichments
+import scala.meta.internal.pc.InlineValueProvider.{Errors => InlineErrors}
+
+class InlineValueSuite extends BaseCodeActionSuite with CommonMtagsEnrichments {
+
+  checkEdit(
+    "inline-local",
+    """|object Main {
+       |  def u(): Unit = {
+       |    val o: Int = 1
+       |    val p: Int = <<o>> + 2
+       |  } 
+       |}""".stripMargin,
+    inlineAll = false,
+    """|object Main {
+       |  def u(): Unit = {
+       |    val p: Int = 1 + 2
+       |  } 
+       |}""".stripMargin,
+  )
+
+  checkEdit(
+    "inline-all-local",
+    """|object Main {
+       |  def u(): Unit = {
+       |    val o: Int = 1
+       |    val p: Int = <<o>> + 2
```

Comment:
Oh, 'cause for those tests it doesn't matter where the cursor is (definition or reference), we have a flag `inlineAll` which is send as a part of the command. So this condition is sorted out on a different level, when code action is created.

Reply:
Ach, right! I missed that. It might be possible to figure out if we are at definition site when invoking the command, but not neccessary. We wouldn't need the additional parameter then.